### ISSUE TITLE
Feature request: Support quote styles

### ISSUE THREAD

**Issue by doctyper** [capaj/vscode-exports-autocomplete#16]
Our style guide at work calls for double quotes, but there's currently no way to set this.

**Comment by flying-sheep** [comment]
for implementation, we should extract `usesStandard` and `eslintCLI` from [`uses-semi.js`](https://github.com/capaj/vscode-exports-autocomplete/blob/master/lib/uses-semi.js) and use both to find the quote style.

**Comment by doctyper** [comment]
Agreed, but I think you should allow a setting to override the check. I think it's better to be explicit than implicit.

**Comment by flying-sheep** [comment]
@capaj why the 😕 face?

**Comment by capaj** [comment]
@flying-sheep who the hell uses backticks for import strings? that's just plain crazy 😈

**Comment by flying-sheep** [comment]
heh, i bet there’s somewhere out there. but it’s not hard to support. shall i make a PR?

**Comment by capaj** [comment]
@flying-sheep I will add the config override. If you can, please extract that bit from `uses-semi.js`

**Comment by flying-sheep** [comment]
OK

**Comment by capaj** [comment]
@doctyper 0.3.3 has been released with the config added. Feel free to try it out and let me know if you have any issues. I will leave this opened, because it would be nice to have the quote style set automagically for each project based on eslint settings.
